vividus-plugin-rest-api/src/main/java/org/vividus/http/transformer/JsonRestApiTableTransformer.java

```diff
@@ -45,7 +47,9 @@ public class JsonRestApiTableTransformer implements ExtendedTableTransformer
     private static final Logger LOGGER = LoggerFactory.getLogger(JsonRestApiTableTransformer.class);
 
     private static final String URL_PROPERTY = "url";
+    @Deprecated(since = "0.5.7", forRemoval = true)
     private static final String VARIABLE_PROPERTY = "variable";
+    private static final String VARIABLE_NAME_PROPERTY = "variableName";
```

Comment:
no docs?

Reply:
proper warning is added into docs on the same way as for `url`